src/linux/listen.rs

```diff
@@ -26,25 +28,17 @@ pub fn listen(callback: Callback) -> Result<(), ListenError> {
         if dpy_control.is_null() || dpy_data.is_null() {
             return Err(ListenError::MissingDisplayError);
         }
-        // Enable synchronization
-        xlib::XSynchronize(dpy_control, 1);
-
-        let extension_name = CString::new("RECORD").unwrap();
 
+        let extension_name = CStr::from_bytes_with_nul(b"RECORD\0").unwrap();
```

Comment:
What is the benefit from that over CString ?

Reply:
This doesn't use heap allocation. Similar to the difference between &str and String.